### DISCUSSION TITLE
ListBoxItem PointerPressed not working

### DISCUSSION THREAD

**Discussion by NemuruYama** [AvaloniaUI/Avalonia#17330] [Q&A] [answered]
### Describe the bug

Essentially, if we have thise code:
`<ListBoxItem PointerPressed="InputElement_OnPointerPressed">Test1</ListBoxItem>`

Nothing every happens.

### To Reproduce

1:
Create a new Avalonia .NET MVVM App

2:
Go to MainWindow.xaml and add a ListBox
```xaml
<ListBox>
    <ListBoxItem PointerPressed="InputElement_OnPointerPressed">Test</ListBoxItem>
    <ListBoxItem PointerPressed="InputElement_OnPointerPressed">Test</ListBoxItem>
    <ListBoxItem PointerPressed="InputElement_OnPointerPressed">Test</ListBoxItem>
</ListBox>
```

3: 
Go to MainWindow.xaml.cs and add `InputElement_OnPointerPressed`
```cs
private void InputElement_OnPointerPressed(object? sender, PointerPressedEventArgs e)
{
    throw new System.NotImplementedException();
}
```

4:
Set a debug point on the Error, but it should also work without.
Then run the application and press on any of the ListBoxItems

### Expected behavior

I would expect it to call the PointPressed event.
I know that it probably has to do with the fact that you are selecting stuff, but for my use case, this is simply not useful.
I need to be capable of detecting when something is pressed and released, and that is impossible at the moment.

### Avalonia version

11.1.0

### OS

Windows

**Comment by IanRawley** [comment, marked as answer]
As you say, OnPointerPressed is handled by the ListBoxItem.  Handlers added from XAML are on the Bubble side, so in this case they'll never fire.  If you want to get the event before it's handled, or even if it is already handled, you'll need to add the handler from code behind using AddHandler() and specifying Tunnel or to fire on handled routed events..
